sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -86,3 +152,163 @@ def version(self):
     @version.setter
     def version(self, value):
         self["version"] = value
+
+    def _get_epk_range_for_prefix_partition_key(self, pk_value: list) -> Range:
+        if self.kind != "MultiHash":
+            raise ValueError("Unsupported partition definition kind for partial key operations")
```

Comment:
Make the error a bit more friendly, something like "The container's partition key definition is defined with Hierarchical Partition Keys"?

Reply:
I added a friendlier error message.